<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>551</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tabUserRegister">
     <attribute name="title">
      <string>User Register</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabUserQuery">
     <attribute name="title">
      <string>User Query</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabBookRegister">
     <attribute name="title">
      <string>Book Register</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabBookQuery">
     <attribute name="title">
      <string>Book Query</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabLoaned">
     <attribute name="title">
      <string>Loaned</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUserName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPwd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLogin">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>89</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBook">
    <property name="title">
     <string>Book</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLoaned">
    <property name="title">
     <string>Loaned</string>
    </property>
   </widget>
   <addaction name="menuUser"/>
   <addaction name="menuBook"/>
   <addaction name="menuLoaned"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
